client/src/index.js

```diff
@@ -1,7 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import Calendar from "./components/Calendar/Calendar";
 
-import App from "./App.js"
+function App() {
```

Comment:
Let's revert this to the old index.js file. You already have the calendar display at the /calendar link, no reason to do it here too, right?